include/stdio.h

```diff
@@ -72,15 +72,15 @@
 #  define CONFIG_LIBC_TMPDIR "/tmp"
 #endif
 
-#define P_tmpdir   CONFIG_LIBC_TMPDIR
+#define p_tmpdir   CONFIG_LIBC_TMPDIR
```

Comment:
we can't change P_tmpdir/L_tmpnam since both is defined by standard:
https://pubs.opengroup.org/onlinepubs/9699919799/functions/tempnam.html

Reply:
But we can add them to nxstyle white list:
https://github.com/apache/incubator-nuttx/blob/master/tools/nxstyle.c#L207

Reply:
thanks for the info